server/app/views/applicant/ApplicantUpsellCreateAccountView.java

```diff
@@ -105,7 +105,9 @@ public Content render(
                 div()
                     .with(
                         TextFormatter.formatText(
-                            customConfirmationMessage.getOrDefault(locale), true))
+                            customConfirmationMessage.getOrDefault(locale),
```

Comment:
Formats custom text used on the confirmation screen for the applicant after they submit an application